sources/core/Stride.Core.Mathematics/Matrix.cs

```diff
@@ -506,6 +528,7 @@ public float Determinant()
         /// Inverts the matrix.
         /// If the matrix cannot be inverted (eg. Determinant was zero), then the matrix will be set equivalent to <see cref="Zero"/>.
         /// </summary>
+        [Obsolete("Use MathUtil.Invert instead. This method will be removed in future versions.")]
```

Comment:
Use an attribute constructor that takes `DiagnosticId`.

```cs
[Obsolete("Use MathUtil.Invert instead.", DiagnosticId = "STRIDE2100")]
public void Invert();
```
or
```cs
[Obsolete("Use MathUtil.Invert instead.", DiagnosticId = "STRIDE2100", UrlFormat = "https://doc.stride3d.net/latest/en/whatever/{0}")]
public void Invert();
```
The diagnostic `STRIDE2000` was used previously, so just reserve the range 2100-2199. Use as few diagnostics codes as possible, but do use a different one if the situation is different - if people would need to supress them in different situations, for example. For example use `STRIDE2101` for deprecations of overloads that take `in`, `ref` or `ref ro` but which are otherwise operations that shouldn't be deprecated such as `Add`. `STRDIAG` is a bad prefix imho. `STR` is string, not Stride, at least that is the automatic context free association. And yeah its a diagnostic, why even bother saying that. Talk to the docs team on Discord (@VaclavElias) to figure out how much effort it is and what the URL format should be. There should be basically only one format for all of Stride.